src/game/server/instagib/round_stats.cpp

```diff
@@ -123,41 +123,53 @@ void IGameController::GetRoundEndStatsStrPsv(char *pBuf, size_t Size)
 	int ScoreLimit = m_GameInfo.m_ScoreLimit;
 	int TimeLimit = m_GameInfo.m_TimeLimit;
 
-	std::optional<std::string> RedClan = std::nullopt;
-	std::optional<std::string> BlueClan = std::nullopt;
+	const char *RedClan = nullptr;
+	const char *BlueClan = nullptr;
 
 	for(const CPlayer *pPlayer : GameServer()->m_apPlayers)
 	{
 		if(!pPlayer)
 			continue;
 
-		if(pPlayer->GetTeam() == TEAM_RED)
+		if(pPlayer->GetTeam() == TEAM_RED) // gof of c++, please forgive me for my sins
 		{
-			if(!RedClan.has_value())
+			if(strlen(Server()->ClientClan(pPlayer->GetCid())) == 0)
```

Comment:
Use the system.cpp provided `str_length` instead of `strlen`. Or just do this

``Server()->ClientClan(pPlayer->GetCid())[0] == '\0'``